openmetadata-ui/src/main/resources/ui/src/components/DashboardVersion/DashboardVersion.component.tsx

```diff
@@ -216,7 +219,9 @@ const DashboardVersion: FC<DashboardVersionProp> = ({
             name={t('label.custom-property-plural')}
           />
         ),
-        children: (
+        children: !entityPermissions.ViewAll ? (
```

Comment:
Custom Property can be shown for ViewBasic I guess
cc: @harshach

Reply:
I added that because the API for fetching the details throws an error with the message no `ViewAll` permission

Reply:
Alright